### ISSUE TITLE
Installing groops on Arch (Manjaro)

### ISSUE THREAD

**Issue by marcianito** [groops-devs/groops#54]
Hi,

I just wanted to share a bug/problem and fix I found while installing groops.

I am on Manjaro (Arch) and so I followed the respective instructions for installation.

## using AUR
I always got an error like `github.com [ip] could not connected: timeout.`

## building from code
I downloaded the code and tried to build myself but got an error concerning file `inputOutput/file.cpp`

## second (sucessful) try with AUR
Investigating a little more I somewhat found the problem. It seems that the AUR package have an insufficient "adress".
The solution is statet here https://github.com/Jguer/yay/issues/1729

So basically what fixed the AUR installation:
`yay -S --editmenu groops-git groopsgui-git`

Then you have to edit the line in PKGBuild that says "source" from
`git://repo-adress` to `git+https://repo-adress`

And installation when smoothly.

**Comment by sestras** [comment]
Hi, thanks for sharing this issue. The AUR GROOPS package was created by @x49, so maybe they can use this information to fix it?

**Comment by x49** [comment]
I updated the groops-git and groopsgui-git AUR packages with the best practices you pointed out, thank you for bringing this to our attention and providing the appropriate fix.
